README.md

```diff
@@ -1,5 +1,9 @@
 # Commit Analyzer
 
+## Summary
```

Comment:
A project should have a summary section to mirror the GitHub description.  This benefits copying the configured brief description to configuration files when you do not have network access.

The ordering can be adjusted, at option; I personally prefer the license information before the complete description but I wanted to first of all discuss this topic with you. :wrench:

Reply:
Hmm, normally I would be against putting the license before the description, but since the license is very short, and it makes sense to have the configured options follow the description, I'd agree with you on this one, to put the license after the summary.

Reply:
:wrench: :heavy_check_mark: